base_layer/key_manager/src/mnemonic.rs

```diff
@@ -171,7 +186,7 @@ fn find_mnemonic_word_from_index(index: usize, language: MnemonicLanguage) -> Re
 }
 
 /// Converts a vector of bytes to a sequence of mnemonic words using the specified language
-pub fn from_bytes(bytes: &[u8], language: MnemonicLanguage) -> Result<Vec<String>, MnemonicError> {
+pub fn from_bytes(bytes: &[u8], language: MnemonicLanguage) -> Result<SeedWords, MnemonicError> {
     let mut bits = bytes_to_bits(bytes);
```

Comment:
Shouldn't `bits` be considered sensitive data?

Reply:
They can be, definitely ! Nice catch